src/cmd/tools/wait.go

```diff
@@ -50,6 +51,13 @@ var waitForCmd = &cobra.Command{
 			condition = args[2]
 		}
 
+		// Check if waitType is JSONPath or condition
+		if utils.IsJSONPathWaitType(condition) {
```

Comment:
If we do combine condition and jsonpath we should make that clear in the jsonschema on the ZarfComponentActionWaitCluster struct since this wouldn't be default kubectl behavior

Reply:
sounds good, updated

Reply:
Looking at this again potentially moving 53 down to the new pkg/utils/wait.go file might be good.